version/1/4/fr/code_of_conduct.md

```diff
@@ -15,7 +15,7 @@ religion ou la nationalité.
 Exemples de comportements qui contribuent à créer un environnement positif :
 
 * l'utilisation d'un langage ouvert et accueillant
-* être respectueux des différents points de vue et expériences vécues
```

Comment:
I can understand you want to remove "lived experiences" to "experiences" only.
This broadens the purpose, but might weaken the wording.
I have no opinion on that, I'll let contributors choose based from my translation.

Reply:
Honestly the whole thing doesn't make sense in any language so I don't think it matters